src/components/account/ResetPassword.test.tsx

```diff
@@ -0,0 +1,51 @@
+import UserService from '@services/user.service'
+import {cleanup, render, screen, waitFor} from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import {REQUEST_RESET_PASSWORD_MESSAGE} from '__test_utils/test_server/userHandlers'
+import {createWrapper} from '__test_utils/utils'
+import {ResetPassword} from './ResetPassword'
+
+export const setUp = () => {
+  const user = userEvent.setup()
+  const spyOnRequestResetPassword = jest.spyOn(UserService, 'requestResetPassword')
+  const element = render(<ResetPassword />, {wrapper: createWrapper()})
+
+  const emailField = screen.getByLabelText('Email')
+  const submitButton = screen.getByRole('button', {name: /reset my password/i})
+
+  return {user, spyOnRequestResetPassword, element, emailField, submitButton}
+}
+
+afterEach(() => {
+  jest.restoreAllMocks()
+  cleanup()
+})
+
+describe('ResetPassword', () => {
+  test('should not be able to request password reset when email is empty', async () => {
+    const {emailField, submitButton, spyOnRequestResetPassword} = setUp()
+    expect(emailField).toHaveTextContent('')
+    expect(submitButton).toBeDisabled()
+    expect(spyOnRequestResetPassword).toHaveBeenCalledTimes(0)
+  })
+
+  test('should show success banner when password reset request is sent', async () => {
+    const {user, emailField, submitButton, spyOnRequestResetPassword} = setUp()
+    const testEmail = 'test@fake.com'
+
+    user.type(emailField, testEmail)
+    await waitFor(() => {
+      expect(emailField).toHaveValue(testEmail)
+    })
+
+    expect(submitButton).not.toBeDisabled()
+    user.click(submitButton)
```

Comment:
Should still `await` the `user` actions

```suggestion
    await user.type(emailField, testEmail)
    await waitFor(() => {
      expect(emailField).toHaveValue(testEmail)
    })

    expect(submitButton).not.toBeDisabled()
    await user.click(submitButton)
```

Reply:
Calling `await` before `click` or `type` causes some `act` warnings to appear -- this seems to be related to testing-library packages using different version of @testing-library/dom, which have different configurations. I've opened a [ticket](https://sagebionetworks.jira.com/browse/DHP-1123) to track the issue.